src/Snippets/Docs/Migration.Bulkhead.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Net.Http;
+
+namespace Snippets.Docs;
+
+internal static partial class Migration
```

Comment:
I like the structure of these migrations. I have two small suggestions:

1. I am not sure if grouping multiple calls into one migration is the best way, because it might confuse those who don't really understand Polly that much and are just doing migration. So that when they look at the code they see that these 4 commands are replaced with 2, how what why? Suddenly there is no async, they might not be sure what to do if they happen to just look at the code. I understand that there will be future notes in the migration guide.
2. I would probably add simple comments to the Migration snippets like they are in Context. It makes the code easier to understand. It can be kind of reassuring that you know the intent behind the lines in the snippet. I am not certain that every line needs it, just a food for thought.

Reply:
Good point I'll try to add comments for more context into the snippets